### PR TITLE
Fix/users time zone

### PR DESCRIPTION
**Pull request by SuzyNakayama1**
Just deleted the time_zone from the requirements in user Scheme.

### CODE REVIEW

**Review by adeshkevich — APPROVED** [review]

File: specs/apis.v1.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "openapi": "3.0.3",
+  "openapi": "3.1.0",
```
Comment (konstantin-smolyakov-idt) [inline specs/apis.v1.json:2]: **question** Why are we doing that? We previously reverted because most tools don't support 3.1.x yet. For example the libraries, that automation QA use to validate the response to match the schemas.

File: specs/apis.v1.json [hunk 2]
```diff
@@ -767,7 +758,6 @@
           "200": {
             "headers": {
               "x-RateLimit-Limit": {
-                "type": "string",
```
Comment (konstantin-smolyakov-idt) [inline specs/apis.v1.json:770]: **question** Why is this deleted?

Comment (SuzyNakayama1) [reply specs/apis.v1.json:770]: That is build automatically, we didn't change anything related to that.

File: specs/apis.v1.json (hunk 1 above)
Comment (SuzyNakayama1) [reply specs/apis.v1.json:2]: @adeshkevich and I had a conversation about the possible impacts of this change, and he mentioned we would have to change a lot and that we should probably leave as is for now.
